manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,22 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    unless $::osfamily == 'RedHat' {
```

Comment:
This might be a bit picky, but in my mind ```if x == y``` is easier to read and if both true and false are present I'd prefer to use if.